t/stub/Sympa/Constants.pm

```diff
@@ -6,7 +6,35 @@
 
 package Sympa::Constants;
 
-use constant LOCALEDIR => 't/locale';
+use constant VERSION => '6.2.52';
+use constant USER    => $ENV{LOGNAME} || $ENV{USER} || getpwuid($<);
+use constant GROUP   => $ENV{LOGNAME} || $ENV{USER} || getgrgid($<);
```

Comment:
- `getpwuid` and `getgrgid` are executed in list context and don't give single name of user/group.
- `$ENV{LOGNAME}` is not an actual user name (e.g. if superuser changed their own user by `su`, login name won't change).
- `$ENV{LOGNAME}` and `$ENV{USER}` are not appropriate for user's group.

Thus IMHO:
``` perl
use constant USER    => $ENV{USER} || [getpwuid($<)]->[0];
use constant GROUP => [getgrgid([getpwnam(USER)]->[3])]->[0];
```

Reply:
Thanks @ikedas !

I tested it and it works as well as before, so I see no reason not to comply to your suggestion.

So I followed your recommendation in commit d911ef8.

Reply:
A remark: on another branch, the build completes only when I use the following:

`use constant USER    => [getpwuid($<)]->[0];`

Just as if `$ENV{USER}` returned an unusable value. Do you think it is necessary?

Reply:
I don't know whether `$ENV{USER}` is useful. It is often set by the shell, but not always.

Reply:
I feel that [getpwuid($<)]->[0] is enough in the case of testing.

Reply:
If it was,
``` perl
use constant USER    => [getpwuid $<]->[0];
use constant GROUP   => [getgrgid $(]->[0];
```
may be enough.

Reply:
OK. I'll fix this. Any other remarks?